### PR TITLE
TER-58 Enable Alter test but skip PKs

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: TER-58 

Before asking for review make sure that:

Table utils: https://github.com/keboola/php-table-backend-utils/pull/88
Common: https://github.com/keboola/php-storage-driver-common/pull/21
TD Driver: https://github.com/keboola/php-storage-driver-teradata/pull/49
BQ Driver: https://github.com/keboola/php-storage-driver-bigquery/pull/27 
Connection: https://github.com/keboola/connection/pull/4120
SAPI: https://github.com/keboola/storage-api-php-client/pull/960

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

**Review by romanbracinik — APPROVED** [review]
ok

File: phpunit.xml.dist [hunk 1]
```diff
@@ -300,8 +300,6 @@
 	<testsuite name="paratest-bigquery">
```
Comment (romanbracinik) [inline phpunit.xml.dist:300]: teraz som si vsimol, ze to tu zapinas pre BQ ale pre TD je to stale vypnute to testovanie alterTable

**Review by romanbracinik — COMMENTED** [review]
viz. ^

**Comment by jirkasemmler** [comment]
image
TD alter cajk @romanbracinik

**Comment by romanbracinik** [comment]
> image TD alter cajk @romanbracinik

Okej tak to merge @jirkasemmler

**Pull request merged**